politeiawww/cmswww.go

```diff
@@ -485,4 +509,6 @@ func (p *politeiawww) setCMSWWWRoutes() {
 		p.handlePayInvoices, permissionAdmin)
 	p.addRoute(http.MethodPost, cms.RouteLineItemPayouts,
 		p.handleLineItemPayouts, permissionAdmin)
+	p.addRoute(http.MethodPost, cms.RouteAdminUserInvoices,
```

Comment:
Should this be a GET request?

Reply:
Yah could be I suppose.  Was debating

Reply:
I don't have strong feelings about it either way.  Looks like the user proposals route on pi is a get request so it would make sense if it was consistent.

Reply:
Cool I made is GET